azurerm/azurerm_management_group/child/main.tf

```diff
@@ -0,0 +1,37 @@
+# Summary: A Management Group as a child of another Management Group
+
+# Documentation: https://www.terraform.io/docs/language/settings/index.html
+terraform {
+  required_version = ">= 0.14.0"
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 2.0"
+    }
+  }
+}
+
+# Documentation: https://www.terraform.io/docs/language/values/variables.html
+variable "azure_subscription_id" {
+  type = string
+}
```

Comment:
This is good. Just to clarify:

Here it works without a default value, because we've taken care of it in: https://github.com/ContainerSolutions/terraform-examples/blob/integration/bin/shared_azurerm.sh
So this variable is the only one for the `azurerm` provider to be passed via environment variables (eg. in CI tests).

Reply:
I see, I'd like to create a similar solution for the other providers, I found out how we could test the other Azure providers, we don't have to leave anything untested :)

Reply:
LGTM!